src/theme/global.scss

```diff
@@ -17,6 +17,12 @@ $global-colors: (
   }
 }
 
+.wallet-info {
```

Comment:
I think wallet info should have its own CSS file

Reply:
Done